examples/powertools-examples-cloudformation/infra/cdk/src/test/java/com/myorg/PowertoolsExamplesCloudformationCdkTest.java

```diff
@@ -0,0 +1,24 @@
+ package com.myorg;
+
+ import software.amazon.awscdk.App;
+ import software.amazon.awscdk.assertions.Template;
+ import java.io.IOException;
+
+ import java.util.HashMap;
+
+ import org.junit.jupiter.api.Test;
+
+ public class PowertoolsExamplesCloudformationCdkTest {
+
+     @Test
+     public void testStack() throws IOException {
+         App app = new App();
+         PowertoolsExamplesCloudformationCdkStack stack = new PowertoolsExamplesCloudformationCdkStack(app, "test");
+
+         Template template = Template.fromStack(stack);
+
+         template.hasResourceProperties("AWS::Lambda::Function", new HashMap<String, Number>() {{
```

Comment:
is this assertion really useful ?
Is this test actually useful ? Not sure in this state... and not sure we need a test for this.

Reply:
Removing this test altogether as it isn't useful